tasks/register_with_sources.yml

```diff
@@ -26,7 +26,7 @@
     client_key: "{{ receptor_config_dir }}/{{ account_dir }}/key.pem"
     method: "POST"
     body:
-      name: "Satellite {{ satellite_uuid }}"
+      name: "{{ satellite_url|urlsplit('hostname') }}"
```

Comment:
Do names of things in sources need to be unique? What if I had two different satellites in two different subnets, but for ease of management they would have the same hostname resolvable only in that network?

Wouldn't it be safer in general to use a variable with fallback to hostname?